packages/backend/src/models/SearchModel/utils/search.ts

```diff
@@ -60,6 +60,18 @@ export function getColumnMatchRegexQuery(
     });
 }
 
+export function getColumnMatchExactQuery(
```

Comment:
I don't think we need this type of abstraction for exact match. 
It is useful for search terms, so we can search across name/description, etc
For the exact match we know the column and the name so I think we just need to update the `GroupModel` `find` to have more filter options like organizationUuid. 

```
        if (filters.name) {
            query = query.where(
                'groups.name',
                filters.name,
            );
        }
```